lnwallet/channel.go

```diff
@@ -6207,13 +6351,13 @@ func (lc *LightningChannel) AddHTLC(htlc *lnwire.UpdateAddHTLC,
 //
 // NOTE: It is okay for sourceRef to be nil when unit testing the wallet.
 func (lc *LightningChannel) addHTLC(htlc *lnwire.UpdateAddHTLC,
-	openKey *models.CircuitKey, buffer BufferType) (uint64, error) {
+	openKey *models.CircuitKey, _ BufferType) (uint64, error) {
 
 	lc.Lock()
 	defer lc.Unlock()
 
-	pd := lc.htlcAddDescriptor(htlc, openKey)
-	if err := lc.validateAddHtlc(pd, buffer); err != nil {
+	pd, err := lc.htlcAddDescriptor(htlc, openKey)
```

Comment:
Looks like we dropped the call to `lc.validateAddHtlc()` here. Perhaps by accident? Sounds like that's still needed.